docs/references/general/grid.md

```diff
@@ -0,0 +1,103 @@
+
+SuperDARN radars routinely change operating frequency. Some operating modes use multiple frequencies, either at the same time, or by alternating between frequencies. As a result, some files from some radars include all records for all frequencies in the same file and other radars separate out the individual frequencies into separate files, designated by a channel letter in the file name:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].grid
+
+See [fitacf](fitacf.md) documentation for more information on this naming convention. 
+
+If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) then the common naming convention is:
+
+> YYYYYMMDD.<3-letter abbreviation>.grid
+
+### Combined GRID files 
+
+After GRID files are produced for seperate radars, they are sometimes further combined with other radars GRID files to meka a GRD file. Similar to GRID file just with multiple radar data included. 
+See [combine_grid](../../user_guide/make_grid.md) on how to make GRD files. 
+
+The common naming convention for 24-hr GRD files is:
+
+> YYYYMMDD.grd  
+
+## Fields
+
+GRID files contain a record that contains scalar and vector fields. 
+
+### Scalars
+
+| Field name       | Units      | Data Type    | Description                                                |
+| :----------      | :-----:    | :-------:    | :---                                                       |
+| *start.yearr*  | **None**   | ***short***  | Start Year of the integration period                       |
+| *start.month*  | **None**   | ***short***  | Start Month number (not padded)  of the integration period |
+| *start.day*  | **None**   | ***short***  | Start Day (not padded) of the integration period           |
+| *start.hour*  | *hrs*      | ***short***  | Start Hour of the integration period                       |
+| *start.minute*  | *min*      | ***short***  | Start Minute of the integration period                     |
+| *start.second*  | *s*        | ***short***  | Start Seconds of the integration period                    |
+| *end.year*    | **None**   | ***short***  | End Year of the integration period                         |
+| *end.month*    | **None**   | ***short***  | End Month number (not padded) of the integration period    |
+| *end.day*    | **None**   | ***short***  | End Day (not padded) of the integration period             |
+| *end.hour*    | *hrs*      | ***short***  | End Hour of the integration period                         |
+| *end.minute*    | *min*      | ***short***  | End Minute of the integration period                       |
+| *end.second*    | *s*        | ***short***  | End Seconds of the integration period                      |
+
+
+### Vectors 
+
+!!! Note
+    Let the number of radars in a given GRID/GRD record be defined as *numstid* and
+    Let the number of vector points in a give GRRID/GRD record be defined as *numv*. 
```

Comment:
I think there's a typo here: 
`GRRID/GRD` should be `GRID/GRD`

Reply:
GRRRRRRRRRRID!! :smiling_imp:
```suggestion
    Let the number of vector points in a give GRID/GRD record be defined as *numv*. 
```